lobsterpy/cli.py

```diff
@@ -224,7 +239,15 @@ def get_parser() -> argparse.ArgumentParser:
             "stylesheets when using --style."
         ),
     )
-    plotting_group.add_argument("--title", type=str, default="", help="Plot title")
+    plotting_group.add_argument(
+        "-sty",
```

Comment:
Wondering what other codes use

Reply:
Will check it out

Reply:
Could not find this in other codes, I think it should be fine